src/lib/Behat/BusinessContext/ContentViewContext.php

```diff
@@ -84,7 +84,7 @@ public function iSeeTitle(string $title): void
     private function verifyItemExistenceInSubItemList(string $itemName, string $itemType, string $containerName, bool $itemShouldExist): void
     {
         $isItemInTable = PageObjectFactory::createPage($this->utilityContext, ContentItemPage::PAGE_NAME, $containerName)
-            ->subItemList->table->isElementInTable($itemName, $itemType);
+            ->subItemList->table->isElementInTable($itemName);
```

Comment:
Can it be done in a way that allows us to keep checking the $itemType?

Reply:
As discussed in private, won't be done in this PR